assets/src/design-system/components/menu/stories/index.js

```diff
@@ -0,0 +1,273 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { action } from '@storybook/addon-actions';
+import { boolean, number, text } from '@storybook/addon-knobs';
+import styled, { css } from 'styled-components';
+import { useState, forwardRef } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { DarkThemeProvider } from '../../../storybookUtils';
+import { DROP_DOWN_ITEM } from '../../menu';
+import { Menu } from '../menu';
+import {
+  basicDropDownOptions,
+  effectChooserOptions,
+  nestedDropDownOptions,
+  reallyLongOptions,
+} from '../../../storybookUtils/sampleData';
+import { getOptions } from '../utils';
+
+export default {
+  title: 'DesignSystem/Components/Menu',
+};
+
+const Container = styled.div`
+  width: ${({ narrow }) => (narrow ? 150 : 400)}px;
```

Comment:
I'd love to extract out `NARROW_WIDTH` and `STANDARD_WIDTH` as consts just so it's very clear what we are setting here.